cmd/nfd-topology-updater/main.go

```diff
@@ -139,6 +142,7 @@ func initFlags(flagset *flag.FlagSet) (*topology.Args, *resourcemonitor.Args) {
 		"Pod Resource Socket path to use.")
 	flagset.StringVar(&args.ConfigFile, "config", "/etc/kubernetes/node-feature-discovery/nfd-topology-updater.conf",
 		"Config file to use.")
+	flagset.StringVar(&args.KubeletDirPath, "kubelet-dir-path", DefaultKubeletDirPath, "Kubelet directory path for watching state and checkpoint files")
```

Comment:
We want to document this in `docs/reference/topology-updater-commandline-reference.md`.

Also, have you checked if we need to update some other parts of the documentation e.g. `docs/usage/nfd-topology-updater.md`?

Reply:
Just did. Thanks